Outstanding TCK Test failures in webservices12 test suite

https://ci.eclipse.org/jakartaee-tck/job/eftl-jakartaeetck-run-900/2/junit-reports-with-handlebars/test-summary/130df8fc-7f00-457d-9d74-80cb1c4791fb.html

We see exception stacktrace during webservices12 tests, with app deployment failure message.

Details of CI run logs - https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck-grao/job/master/11/testReport/junit/com.sun.ts.tests.webservices12.wsdlImport.file.nested2/Client/jakartaeetck_run___webservices12___InvokeMethod_from_wsappclient/

Details of the CI run - https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck-grao/job/master/11/testReport/

Glassfish logs for test can be downloaded here - https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck-grao/job/master/11/artifact/webservices12-results.tar.gz

Exception stack trace is as follows:
-----------------------------------------------------------------------------------------------------------------------------
[2020-08-02T09:34:57.396+0000] [glassfish 6.0] [SEVERE] [] [jakarta.enterprise.system.tools.deployment.common] [tid: _ThreadID=84 _ThreadName=AutoDeployer] [timeMillis: 1596360897396] [levelValue: 1000] [[
  Exception while invoking class org.glassfish.webservices.WebServicesDeployer load method
java.lang.RuntimeException: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],2003533350)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=2044111301
	reified=false)

	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:750)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:103)
	at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:182)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:289)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:472)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:195)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:467)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:516)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:512)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:511)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:542)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:534)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:533)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1441)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:86)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1823)
	at org.glassfish.deployment.autodeploy.AutoOperation.run(AutoOperation.java:140)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deploy(AutoDeployer.java:573)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deployAll(AutoDeployer.java:460)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:388)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:379)
	at org.glassfish.deployment.autodeploy.AutoDeployService$1.run(AutoDeployService.java:209)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],2003533350)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=2044111301
	reified=false)

	at org.jvnet.hk2.internal.Collector.throwIfErrors(Collector.java:65)
	at org.jvnet.hk2.internal.Utilities.handleErrors(Utilities.java:363)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1185)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.access$400(ServiceLocatorImpl.java:107)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1166)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1163)
	at org.glassfish.hk2.utilities.cache.internal.WeakCARCacheImpl.compute(WeakCARCacheImpl.java:105)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetDescriptor(ServiceLocatorImpl.java:1246)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:750)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:717)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:687)
	at org.glassfish.webservices.WebServicesDeployer.copyExtraFilesToGeneratedFolder(WebServicesDeployer.java:838)
	at org.glassfish.webservices.WebServicesDeployer.populateWsdlFilesForPublish(WebServicesDeployer.java:770)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:748)
	... 27 more
Caused by: com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:193)
	at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:54)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2234)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.narrow(ServiceLocatorImpl.java:2296)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1172)
	... 38 more
Caused by: org.osgi.framework.BundleException: Unable to resolve org.glassfish.main.appclient.gf-client-module [80](R 80.0): missing requirement [org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent) Unresolved requirements: [[org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.startBundle(OSGiModuleImpl.java:224)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.access$000(OSGiModuleImpl.java:57)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl$2.run(OSGiModuleImpl.java:177)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:174)
	... 43 more
]]

[2020-08-02T09:34:57.397+0000] [glassfish 6.0] [SEVERE] [NCLS-CORE-00026] [jakarta.enterprise.system.core] [tid: _ThreadID=84 _ThreadName=AutoDeployer] [timeMillis: 1596360897397] [levelValue: 1000] [[
  Exception during lifecycle processing
java.lang.RuntimeException: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],2003533350)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=2044111301
	reified=false)

	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:750)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:103)
	at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:182)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:289)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:472)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:195)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:467)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:516)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:512)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:511)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:542)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:534)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:533)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1441)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:86)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1823)
	at org.glassfish.deployment.autodeploy.AutoOperation.run(AutoOperation.java:140)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deploy(AutoDeployer.java:573)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deployAll(AutoDeployer.java:460)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:388)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:379)
	at org.glassfish.deployment.autodeploy.AutoDeployService$1.run(AutoDeployService.java:209)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],2003533350)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=2044111301
	reified=false)

	at org.jvnet.hk2.internal.Collector.throwIfErrors(Collector.java:65)
	at org.jvnet.hk2.internal.Utilities.handleErrors(Utilities.java:363)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1185)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.access$400(ServiceLocatorImpl.java:107)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1166)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1163)
	at org.glassfish.hk2.utilities.cache.internal.WeakCARCacheImpl.compute(WeakCARCacheImpl.java:105)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetDescriptor(ServiceLocatorImpl.java:1246)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:750)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:717)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:687)
	at org.glassfish.webservices.WebServicesDeployer.copyExtraFilesToGeneratedFolder(WebServicesDeployer.java:838)
	at org.glassfish.webservices.WebServicesDeployer.populateWsdlFilesForPublish(WebServicesDeployer.java:770)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:748)
	... 27 more
Caused by: com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:193)
	at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:54)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2234)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.narrow(ServiceLocatorImpl.java:2296)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1172)
	... 38 more
Caused by: org.osgi.framework.BundleException: Unable to resolve org.glassfish.main.appclient.gf-client-module [80](R 80.0): missing requirement [org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent) Unresolved requirements: [[org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.startBundle(OSGiModuleImpl.java:224)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.access$000(OSGiModuleImpl.java:57)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl$2.run(OSGiModuleImpl.java:177)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:174)
	... 43 more
]]

[2020-08-02T09:34:57.397+0000] [glassfish 6.0] [SEVERE] [] [jakarta.enterprise.system.core] [tid: _ThreadID=84 _ThreadName=AutoDeployer] [timeMillis: 1596360897397] [levelValue: 1000] [[
  Exception while loading the app]]

[2020-08-02T09:34:57.412+0000] [glassfish 6.0] [SEVERE] [] [jakarta.enterprise.system.core] [tid: _ThreadID=84 _ThreadName=AutoDeployer] [timeMillis: 1596360897412] [levelValue: 1000] [[
  Exception while loading the app : A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],2003533350)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=2044111301
	reified=false)

MultiException stack 1 of 2
com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:193)
	at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:54)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2234)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.narrow(ServiceLocatorImpl.java:2296)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1172)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.access$400(ServiceLocatorImpl.java:107)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1166)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1163)
	at org.glassfish.hk2.utilities.cache.internal.WeakCARCacheImpl.compute(WeakCARCacheImpl.java:105)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetDescriptor(ServiceLocatorImpl.java:1246)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:750)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:717)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:687)
	at org.glassfish.webservices.WebServicesDeployer.copyExtraFilesToGeneratedFolder(WebServicesDeployer.java:838)
	at org.glassfish.webservices.WebServicesDeployer.populateWsdlFilesForPublish(WebServicesDeployer.java:770)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:748)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:103)
	at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:182)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:289)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:472)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:195)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:467)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:516)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:512)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:511)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:542)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:534)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:533)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1441)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:86)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1823)
	at org.glassfish.deployment.autodeploy.AutoOperation.run(AutoOperation.java:140)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deploy(AutoDeployer.java:573)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deployAll(AutoDeployer.java:460)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:388)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:379)
	at org.glassfish.deployment.autodeploy.AutoDeployService$1.run(AutoDeployService.java:209)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: org.osgi.framework.BundleException: Unable to resolve org.glassfish.main.appclient.gf-client-module [80](R 80.0): missing requirement [org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent) Unresolved requirements: [[org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.startBundle(OSGiModuleImpl.java:224)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.access$000(OSGiModuleImpl.java:57)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl$2.run(OSGiModuleImpl.java:177)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:174)
	... 43 more
MultiException stack 2 of 2
java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],2003533350)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=2044111301
	reified=false)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2243)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.narrow(ServiceLocatorImpl.java:2296)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1172)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.access$400(ServiceLocatorImpl.java:107)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1166)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1163)
	at org.glassfish.hk2.utilities.cache.internal.WeakCARCacheImpl.compute(WeakCARCacheImpl.java:105)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetDescriptor(ServiceLocatorImpl.java:1246)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:750)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:717)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:687)
	at org.glassfish.webservices.WebServicesDeployer.copyExtraFilesToGeneratedFolder(WebServicesDeployer.java:838)
	at org.glassfish.webservices.WebServicesDeployer.populateWsdlFilesForPublish(WebServicesDeployer.java:770)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:748)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:103)
	at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:182)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:289)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:472)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:195)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:467)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:516)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:512)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:511)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:542)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:534)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:533)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1441)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:86)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1823)
	at org.glassfish.deployment.autodeploy.AutoOperation.run(AutoOperation.java:140)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deploy(AutoDeployer.java:573)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deployAll(AutoDeployer.java:460)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:388)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:379)
	at org.glassfish.deployment.autodeploy.AutoDeployService$1.run(AutoDeployService.java:209)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
]]
-------------------------------------------------------------------------------------------------------------------------------

I'm looking at one of the [failures from last nights run for com.sun.ts.tests.webservices12.sec.descriptors.ejb.basicSSL.Client](https://ci.eclipse.org/jakartaee-tck/job/jakartaeetck-nightly-run-master/72/junit-reports-with-handlebars/test-summary/0a8985f7-0abf-4aa1-8e0d-f7b11b951ef0.html) that also gets a NPE:
```
08-10-2020 19:57:21:  SVR-ERROR: javax.naming.NamingException: Lookup failed for 'java:comp/env/service/wssecEjb/basicNoId' in SerialContext[myEnv={java.naming.factory.state=com.sun.corba.ee.impl.presentation.rmi.JNDIStateFactoryImpl, java.naming.factory.url.pkgs=com.sun.enterprise.naming} [Root exception is javax.naming.NamingException [Root exception is java.lang.NullPointerException]]
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:467)
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:414)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at javax.naming.InitialContext.lookup(InitialContext.java:417)
	at com.sun.ts.tests.webservices12.sec.descriptors.ejb.basicSSL.Client.getPorts(Client.java:73)
	at com.sun.ts.tests.webservices12.sec.descriptors.ejb.basicSSL.Client.setup(Client.java:149)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.ts.lib.harness.EETest.run(EETest.java:569)
	at com.sun.ts.lib.harness.ServiceEETest.run(ServiceEETest.java:115)
	at com.sun.ts.tests.common.vehicle.wsejb.WSEJBVehicle.runTest(WSEJBVehicle.java:93)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.glassfish.ejb.security.application.EJBSecurityManager$5.run(EJBSecurityManager.java:1134)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.glassfish.ejb.security.application.EJBSecurityManager.doAsPrivileged(EJBSecurityManager.java:1032)
	at org.glassfish.ejb.security.application.EJBSecurityManager.invoke(EJBSecurityManager.java:1139)
	at com.sun.ejb.containers.BaseContainer.invokeTargetBeanMethod(BaseContainer.java:4220)
	at com.sun.ejb.containers.BaseContainer.__intercept(BaseContainer.java:4775)
	at com.sun.ejb.containers.BaseContainer.intercept(BaseContainer.java:4755)
	at com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:173)
	at com.sun.ejb.containers.EJBObjectInvocationHandler.invoke(EJBObjectInvocationHandler.java:91)
	at com.sun.proxy.$Proxy366.runTest(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie.dispatchToMethod(ReflectiveTie.java:113)
	at com.sun.corba.ee.impl.presentation.rmi.ReflectiveTie._invoke(ReflectiveTie.java:143)
	at com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatchToServant(ServerRequestDispatcherImpl.java:492)
	at com.sun.corba.ee.impl.protocol.ServerRequestDispatcherImpl.dispatch(ServerRequestDispatcherImpl.java:163)
	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequestRequest(MessageMediatorImpl.java:1520)
	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequest(MessageMediatorImpl.java:1396)
	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleInput(MessageMediatorImpl.java:901)
	at com.sun.corba.ee.impl.protocol.giopmsgheaders.RequestMessage_1_2.callback(RequestMessage_1_2.java:183)
	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.handleRequest(MessageMediatorImpl.java:665)
	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.dispatch(MessageMediatorImpl.java:467)
	at com.sun.corba.ee.impl.protocol.MessageMediatorImpl.doWork(MessageMediatorImpl.java:2193)
	at com.sun.corba.ee.impl.threadpool.ThreadPoolImpl$WorkerThread.performWork(ThreadPoolImpl.java:467)
	at com.sun.corba.ee.impl.threadpool.ThreadPoolImpl$WorkerThread.run(ThreadPoolImpl.java:510)
Caused by: javax.naming.NamingException [Root exception is java.lang.NullPointerException]
	at org.glassfish.webservices.WebServiceReferenceManagerImpl.resolveWSReference(WebServiceReferenceManagerImpl.java:205)
	at com.sun.enterprise.container.common.impl.ComponentEnvManagerImpl$WebServiceRefProxy.create(ComponentEnvManagerImpl.java:944)
	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:721)
	at com.sun.enterprise.naming.impl.GlassfishNamingManagerImpl.lookup(GlassfishNamingManagerImpl.java:691)
	at com.sun.enterprise.naming.impl.JavaURLContext.lookup(JavaURLContext.java:135)
	at com.sun.enterprise.naming.impl.SerialContext.lookup(SerialContext.java:447)
	... 43 more
Caused by: java.lang.NullPointerException
	at org.glassfish.webservices.WebServiceReferenceManagerImpl.getServletAdapterBasedOnWsdlUrl(WebServiceReferenceManagerImpl.java:476)
	at org.glassfish.webservices.WebServiceReferenceManagerImpl.getServletAdapter(WebServiceReferenceManagerImpl.java:397)
	at org.glassfish.webservices.WebServiceReferenceManagerImpl.generateWsdlFile(WebServiceReferenceManagerImpl.java:335)
	at org.glassfish.webservices.WebServiceReferenceManagerImpl.initiateInstance(WebServiceReferenceManagerImpl.java:263)
	at org.glassfish.webservices.WebServiceReferenceManagerImpl.resolveWSReference(WebServiceReferenceManagerImpl.java:111)
```
While looking at the test source in jakartaee-tck/src/com/sun/ts/tests/webservices12/sec/descriptors/ejb/basicSSL, I opened *.xml and noticed that WSSecEjbBasicSSL_ejb.jar.sun-ejb-jar.xml is referencing http://www.sun.com/software/appserver/dtds/sun-ejb-jar_3_1-0.dtd which does support jax-rpc although stub-property/call-property aren't used, I wonder if the use of sun-ejb-jar_3_1-0.dtd could be the cause of problems we are seeing (in that jax-rpc classes cannot be resolved by deployment classes trying to deal with sun-ejb-jar_3_1-0.dtd).  


sun-ejb-jar_3_1-0.dtd also seems to be in tests:
```
jacc/ejb/mr/jacc_ejb_mr_ejblitesecuredjsp_vehicle_web.war.sun-ejb-jar.xml
jws/common/xml/deploymentDescriptors/templates/APPNAME_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/ee20/cditests/mdb/cditestsmdb_ejb.jar.sun-ejb-jar.xml
jms/ee20/cditests/ejbweb/cditestsejbweb_ejb.jar.sun-ejb-jar.xml
jms/ee20/cditests/usecases/cditestsusecases_ejb.jar.sun-ejb-jar.xml
jms/ee/ejb/sessionQtests/jms_ejb_sessionQtests_ejb.jar.sun-ejb-jar.xml
jms/ee/ejb/sessionTtests/jms_ejb_sessionTtests_ejb.jar.sun-ejb-jar.xml
jms/ee/ejb/queueCMTTests/jms_ejb_queueCMTTests_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgTypesT3/mdb_msgTypesT3_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_synchrec/mdb_synchrec_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_rec/mdb_asynch_receives_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgHdrQ/mdb_msgHdrQ_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgTypesT2/mdb_msgTypesT2_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_sndQ/mdb_sndQ_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgPropsQ/mdb_msgPropsQ_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgPropsT/mdb_msgPropsT_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/xa/mdb_msg_xa_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_sndToTopic/mdb_sndToTopic_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgTypesT1/mdb_msgTypesT1_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_exceptQ/mdb_exceptQ_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgTypesQ3/mdb_msgTypesQ3_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgTypesQ1/mdb_msgTypesQ1_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_sndToQueue/mdb_sndToQueue_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgHdrT/mdb_msgHdrT_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_msgTypesQ2/mdb_msgTypesQ2_ejb.jar.sun-ejb-jar.xml
jms/ee/mdb/mdb_exceptT/mdb_exceptT_ejb.jar.sun-ejb-jar.xml
jms/ee/ejbweb/xa/transaction_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/jmscontexttopictests/jmscontexttopictests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/messageproducertests/messageproducertests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/sessiontests/sessiontests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/jmsconsumertests/jmsconsumertests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/connectionfactorytests/connectionfactorytests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/jmsproducerqueuetests/jmsproducerqueuetests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/jmscontextqueuetests/jmscontextqueuetests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core20/jmsproducertopictests/jmsproducertopictests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/bytesMsgTopic/bytesMsgTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/objectMsgQueue/objectMsgQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/exceptionTopic/exceptionTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/foreignMsgTopic/foreignMsgTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/queueConnection/queueConnection_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/queueMsgHeaders/queueMsgHeaders_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/exceptionQueue/exceptionQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/bytesMsgQueue/bytesMsgQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/topicConnection/topicConnection_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/sessiontests/sessiontests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedTopicPublisher/closedTopicPublisher_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedTopicConnection/closedTopicConnection_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/streamMsgQueue/streamMsgQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedQueueConnection/closedQueueConnection_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/topictests/topictests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/messageTopic/messageTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/selectorTopic/selectorTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/topicMsgProperties/topicMsgProperties_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/streamMsgTopic/streamMsgTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/mapMsgQueue/mapMsgQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedQueueSender/closedQueueSender_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedQueueSession/closedQueueSession_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/mapMsgTopic/mapMsgTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/queueMsgProperties/queueMsgProperties_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/selectorQueue/selectorQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/topicMsgHeaders/topicMsgHeaders_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/messageProducer/messageProducer_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/foreignMsgQueue/foreignMsgQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedQueueReceiver/closedQueueReceiver_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/queuetests/queuetests_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedTopicSubscriber/closedTopicSubscriber_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/objectMsgTopic/objectMsgTopic_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/closedTopicSession/closedTopicSession_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/core/messageQueue/messageQueue_ejb_vehicle_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msg_xa_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_sndToTopic_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgPropsT_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgTypesT1_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgTypesQ2_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgPropsQ_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_sndToQueue_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_sndQ_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_synchrec_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgTypesQ3_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgTypesQ1_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgHdrT_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgTypesT3_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_asynch_receives_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgHdrQ_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/cditestsmdb_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_exceptQ_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_msgTypesT2_ejb.jar.sun-ejb-jar.xml
jms/commonee/xml/descriptors/genericra/mdb_exceptT_ejb.jar.sun-ejb-jar.xml
webservices12/deploy/jarDeploy/WSJarDeployApp_component.jar.sun-ejb-jar.xml
webservices12/deploy/portcomplink/ejb/WSPortCompLinkEjb_ejb.jar.sun-ejb-jar.xml
webservices12/servlet/WSMTOMSBFullDDsTest/WSMTOMSBFullDDsTest_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
webservices12/servlet/WebServiceRefsTest/client/WebServiceRefsTest_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
webservices12/sec/descriptors/servlet/basicSSL/WSSecWarBasicSSL_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
webservices12/sec/descriptors/ejb/basicSSL/WSSecEjbBasicSSL_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
webservices12/sec/descriptors/ejb/basicSSL/WSSecEjbBasicSSL_ejb.jar.sun-ejb-jar.xml
webservices12/sec/descriptors/ejb/certificate/WSSecEjbCertificate_ejb.jar.sun-ejb-jar.xml
webservices12/sec/annotations/ejb/clientcert/Hello_ejb.jar.sun-ejb-jar.xml
webservices12/sec/annotations/ejb/basicauth/Hello_ejb.jar.sun-ejb-jar.xml
webservices12/sec/annotations/ejb/basicauthssl/Hello_ejb.jar.sun-ejb-jar.xml
webservices12/narrow/WSNarrow_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/descriptors/WSEjbOverrideWSRefWithDDsTest/WSEjbOverrideWSRefWithDDsTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/descriptors/WSEjbOverrideWSRefHCWithDDsTest/WSEjbOverrideWSRefHCWithDDsTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbMultipleClientInjectionTest2/WSEjbMultipleClientInjectionTest2_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbSOAPHandlersTest/WSEjbSOAPHandlersTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbPortMethodInjectionTest/WSEjbPortMethodInjectionTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbWebServiceRefTest2/WSEjbWebServiceRefTest2_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbPortFieldInjectionTest/WSEjbPortFieldInjectionTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbWebServiceRefTest1/WSEjbWebServiceRefTest1_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbNoWebServiceRefInClientTest/WSEjbNoWebServiceRefInClientTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbSOAPHandlersTest2/WSEjbSOAPHandlersTest2_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbAsyncTest/server/WSEjbAsyncTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbWebServiceProviderTest/WSEjbWebServiceProviderTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/HandlerTest/server/WSHandlerTestEJB_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbPkgEPAndClientInSameEarTest/WSEjbPkgEPAndClientInSameEarTest_ejb.jar.sun-ejb-jar.xml
webservices12/ejb/annotations/WSEjbMultipleClientInjectionTest1/WSEjbMultipleClientInjectionTest1_ejb.jar.sun-ejb-jar.xml
jpa/ee/packaging/ejb/standalone/jpa_ee_packaging_ejb_standalone_component_ejb.jar.sun-ejb-jar.xml
jpa/ee/packaging/ejb/exclude/jpa_ee_packaging_ejb.jar.sun-ejb-jar.xml
jpa/ee/packaging/ejb/descriptor/jpa_ee_packaging_ejb_descriptor_ejb.jar.sun-ejb-jar.xml
jpa/ee/packaging/ejb/resource_local/jpa_ee_packaging_ejb_resource_local_ejb.jar.sun-ejb-jar.xml
jpa/ee/propagation/am/jpa_ee_propagation_am_ejb.jar.sun-ejb-jar.xml
jpa/ee/propagation/cm/jta/jpa_ee_propagation_cm_jta_ejb.jar.sun-ejb-jar.xml
jpa/ee/propagation/cm/extended/jpa_ee_propagation_cm_ext_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSRespBindAndAddressingTestUsingAnnotations/WSRespBindAndAddressingTestUsingAnnotationsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSMTOMFeaturesTestUsingAnnotations/WSMTOMFeaturesTestUsingAnnotationsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSAddressingFeaturesTestUsingAnnotations/WSAddressingFeaturesTestUsingAnnotationsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSWebServiceRefLookupDDs/client/WSWSRefLookupDDsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSMTOMFeaturesTestUsingDDs/WSMTOMFeaturesTestUsingDDsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSRespBindAndAddressingTestUsingDDs/WSRespBindAndAddressingTestUsingDDsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/servlet/WSAddressingFeaturesTestUsingDDs/WSAddressingFeaturesTestUsingDDsClnt_ejb.jar.sun-ejb-jar.xml
webservices13/ejb/annotations/WSEjbWSRefRespBindAndAddressingCombinedTest/WSEjbWSRefRespBindAndAddressingCombinedTest_ejb.jar.sun-ejb-jar.xml
webservices13/ejb/annotations/WSEjbWSRefRespBindAndAddressingCombinedTest/WSEjbWSRefRespBindAndAddressingCombinedTestClnt_ejb.jar.sun-ejb-jar.xml
webservices13/ejb/annotations/WSEjbSingletonTest/WSEjbSingletonTest_ejb.jar.sun-ejb-jar.xml
webservices13/ejb/annotations/WSEjbWSRefAndAddressingCombinedTest/WSEjbWSRefAndAddressingCombinedTestClnt_ejb.jar.sun-ejb-jar.xml
webservices13/ejb/annotations/WSEjbWSRefAndAddressingCombinedTest/WSEjbWSRefAndAddressingCombinedTest_ejb.jar.sun-ejb-jar.xml
concurrency/spec/ManagedScheduledExecutorService/managed/forbiddenapi/forbiddenapiTest_ejb.jar.sun-ejb-jar.xml
concurrency/spec/ManagedScheduledExecutorService/inheritedapi/inheritedapi_ejb.jar.sun-ejb-jar.xml
concurrency/spec/ManagedScheduledExecutorService/inheritedapi/counter_ejb.jar.sun-ejb-jar.xml
concurrency/spec/ManagedExecutorService/managed/forbiddenapi/forbiddenapiTest_ejb.jar.sun-ejb-jar.xml
jaxws/wsa/j2w/document/literal/addressingfeature/WSAJ2WDLAddressingFeatureTest_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/wsa/j2w/document/literal/anonymous/WSAJ2WDLAnonymousTest_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/api/jakarta_xml_ws/WebServiceContext/WSWebServiceContext_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/ee/w2j/rpc/literal/sec/secbasic/WSW2JBAT_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/ee/w2j/rpc/literal/httptest/WSW2JRLHttpTest_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/jaxws23/wsa/j2w/document/literal/anonymous/WSAJ2WDLAnonymousTest23_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/httptemplates/ejbendpointnumservices2/APPNAME_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/httptemplates/ejbendpoint/APPNAME_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/httptemplates/clientnumservices2/APPNAME_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/httptemplates/client/APPNAME_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/filetemplates/ejbendpointnumservices2/APPNAME_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/filetemplates/ejbendpoint/APPNAME_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/filetemplates/clientnumservices2/APPNAME_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml
jaxws/common/xml/deploymentDescriptors/filetemplates/client/APPNAME_wsejb_vehicle_ejb.jar.sun-ejb-jar.xml

```

@lukasj  @scottmarlow @smillidge 
Latest run of webservices12 testsuite - https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/968/testReport/
com.sun.ts.tests.webservices12.wsdlImport.file* tests fail with following stacktrace -
----------------------------------------------------------------------------------------------------------------------------------------------------------
[2020-08-20T08:55:45.702+0000] [glassfish 6.0] [SEVERE] [] [jakarta.enterprise.system.tools.deployment.common] [tid: _ThreadID=81 _ThreadName=AutoDeployer] [timeMillis: 1597913745702] [levelValue: 1000] [[
  Exception while invoking class org.glassfish.webservices.WebServicesDeployer load method
java.lang.RuntimeException: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],508976105)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=615311064
	reified=false)

	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:750)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:103)
	at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:182)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:289)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:472)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:195)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:467)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:516)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:512)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:511)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:542)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:534)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:533)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1441)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:86)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1823)
	at org.glassfish.deployment.autodeploy.AutoOperation.run(AutoOperation.java:140)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deploy(AutoDeployer.java:573)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deployAll(AutoDeployer.java:460)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:388)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:379)
	at org.glassfish.deployment.autodeploy.AutoDeployService$1.run(AutoDeployService.java:209)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],508976105)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=615311064
	reified=false)

	at org.jvnet.hk2.internal.Collector.throwIfErrors(Collector.java:65)
	at org.jvnet.hk2.internal.Utilities.handleErrors(Utilities.java:363)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1185)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.access$400(ServiceLocatorImpl.java:107)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1166)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1163)
	at org.glassfish.hk2.utilities.cache.internal.WeakCARCacheImpl.compute(WeakCARCacheImpl.java:105)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetDescriptor(ServiceLocatorImpl.java:1246)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:750)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:717)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:687)
	at org.glassfish.webservices.WebServicesDeployer.copyExtraFilesToGeneratedFolder(WebServicesDeployer.java:838)
	at org.glassfish.webservices.WebServicesDeployer.populateWsdlFilesForPublish(WebServicesDeployer.java:770)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:748)
	... 27 more
Caused by: com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:193)
	at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:54)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2234)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.narrow(ServiceLocatorImpl.java:2296)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1172)
	... 38 more
Caused by: org.osgi.framework.BundleException: Unable to resolve org.glassfish.main.appclient.gf-client-module [80](R 80.0): missing requirement [org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent) Unresolved requirements: [[org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.startBundle(OSGiModuleImpl.java:224)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.access$000(OSGiModuleImpl.java:57)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl$2.run(OSGiModuleImpl.java:177)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:174)
	... 43 more
]]

[2020-08-20T08:55:45.702+0000] [glassfish 6.0] [SEVERE] [NCLS-CORE-00026] [jakarta.enterprise.system.core] [tid: _ThreadID=81 _ThreadName=AutoDeployer] [timeMillis: 1597913745702] [levelValue: 1000] [[
  Exception during lifecycle processing
java.lang.RuntimeException: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],508976105)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=615311064
	reified=false)

	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:750)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:103)
	at org.glassfish.internal.data.ModuleInfo.load(ModuleInfo.java:182)
	at org.glassfish.internal.data.ApplicationInfo.load(ApplicationInfo.java:289)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:472)
	at com.sun.enterprise.v3.server.ApplicationLifecycle.deploy(ApplicationLifecycle.java:195)
	at org.glassfish.deployment.admin.DeployCommand.execute(DeployCommand.java:467)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:516)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2$1.run(CommandRunnerImpl.java:512)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$2.execute(CommandRunnerImpl.java:511)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:542)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$3.run(CommandRunnerImpl.java:534)
	at java.security.AccessController.doPrivileged(Native Method)
	at javax.security.auth.Subject.doAs(Subject.java:360)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:533)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.doCommand(CommandRunnerImpl.java:1441)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl.access$1300(CommandRunnerImpl.java:86)
	at com.sun.enterprise.v3.admin.CommandRunnerImpl$ExecutionContext.execute(CommandRunnerImpl.java:1823)
	at org.glassfish.deployment.autodeploy.AutoOperation.run(AutoOperation.java:140)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deploy(AutoDeployer.java:573)
	at org.glassfish.deployment.autodeploy.AutoDeployer.deployAll(AutoDeployer.java:460)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:388)
	at org.glassfish.deployment.autodeploy.AutoDeployer.run(AutoDeployer.java:379)
	at org.glassfish.deployment.autodeploy.AutoDeployService$1.run(AutoDeployService.java:209)
	at java.util.TimerThread.mainLoop(Timer.java:555)
	at java.util.TimerThread.run(Timer.java:505)
Caused by: A MultiException has 2 exceptions.  They are:
1. com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
2. java.lang.IllegalStateException: Could not load descriptor SystemDescriptor(
	implementation=org.glassfish.appclient.common.ACCAppClientArchivist
	contracts={org.glassfish.appclient.common.ACCAppClientArchivist,com.sun.enterprise.deployment.archivist.Archivist}
	scope=org.glassfish.hk2.api.PerLookup
	qualifiers={}
	descriptorType=CLASS
	descriptorVisibility=NORMAL
	metadata=Bundle-SymbolicName={org.glassfish.main.appclient.gf-client-module},Bundle-Version={6.0.0.SNAPSHOT}
	rank=0
	loader=OsgiPopulatorPostProcessor.HK2Loader(OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW],508976105)
	proxiable=null
	proxyForSameScope=null
	analysisName=null
	id=47
	locatorId=0
	identityHashCode=615311064
	reified=false)

	at org.jvnet.hk2.internal.Collector.throwIfErrors(Collector.java:65)
	at org.jvnet.hk2.internal.Utilities.handleErrors(Utilities.java:363)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1185)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.access$400(ServiceLocatorImpl.java:107)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1166)
	at org.jvnet.hk2.internal.ServiceLocatorImpl$8.compute(ServiceLocatorImpl.java:1163)
	at org.glassfish.hk2.utilities.cache.internal.WeakCARCacheImpl.compute(WeakCARCacheImpl.java:105)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetDescriptor(ServiceLocatorImpl.java:1246)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:750)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.internalGetService(ServiceLocatorImpl.java:717)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.getService(ServiceLocatorImpl.java:687)
	at org.glassfish.webservices.WebServicesDeployer.copyExtraFilesToGeneratedFolder(WebServicesDeployer.java:838)
	at org.glassfish.webservices.WebServicesDeployer.populateWsdlFilesForPublish(WebServicesDeployer.java:770)
	at org.glassfish.webservices.WebServicesDeployer.load(WebServicesDeployer.java:748)
	... 27 more
Caused by: com.sun.enterprise.module.ResolveError: Failed to start OSGiModuleImpl:: Bundle = [org.glassfish.main.appclient.gf-client-module [80]], State = [NEW]
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:193)
	at org.jvnet.hk2.osgiadapter.OsgiPopulatorPostProcessor$1.loadClass(OsgiPopulatorPostProcessor.java:54)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.loadClass(ServiceLocatorImpl.java:2234)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.reifyDescriptor(ServiceLocatorImpl.java:433)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.narrow(ServiceLocatorImpl.java:2296)
	at org.jvnet.hk2.internal.ServiceLocatorImpl.igdCacheCompute(ServiceLocatorImpl.java:1172)
	... 38 more
Caused by: org.osgi.framework.BundleException: Unable to resolve org.glassfish.main.appclient.gf-client-module [80](R 80.0): missing requirement [org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent) Unresolved requirements: [[org.glassfish.main.appclient.gf-client-module [80](R 80.0)] osgi.wiring.package; (osgi.wiring.package=org.glassfish.appclient.client.acc.agent)]
	at org.apache.felix.framework.Felix.resolveBundleRevision(Felix.java:4368)
	at org.apache.felix.framework.Felix.startBundle(Felix.java:2281)
	at org.apache.felix.framework.BundleImpl.start(BundleImpl.java:998)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.startBundle(OSGiModuleImpl.java:224)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.access$000(OSGiModuleImpl.java:57)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl$2.run(OSGiModuleImpl.java:177)
	at java.security.AccessController.doPrivileged(Native Method)
	at org.jvnet.hk2.osgiadapter.OSGiModuleImpl.start(OSGiModuleImpl.java:174)
	... 43 more
]]

[2020-08-20T08:55:45.702+0000] [glassfish 6.0] [SEVERE] [] [jakarta.enterprise.system.core] [tid: _ThreadID=81 _ThreadName=AutoDeployer] [timeMillis: 1597913745702] [levelValue: 1000] [[
  Exception while loading the app]]
---------------------------------------------------------------------------------------------------------------------

Results and logs can be download from - https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/968/artifact/webservices12-results.tar.gz
Inside archive  "/root/vi/glassfish6/glassfish/domains/domain1/logs/" has glassfish logs.

--- a/appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/BundleDescriptor.java
+++ b/appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/BundleDescriptor.java
@@ -643,7 +643,9 @@ public abstract class BundleDescriptor extends RootDeploymentDescriptor implemen
         return DEPLOYMENT_DESCRIPTOR_DIR;
     }
 
-
+    public String getRawModuleID() {
+        return moduleID;
+    }
     /**
      * @return the wsdl directory location inside the archive file
      */

--- a/appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WsUtil.java
+++ b/appserver/webservices/jsr109-impl/src/main/java/org/glassfish/webservices/WsUtil.java
@@ -544,9 +544,21 @@ public class WsUtil {
                                     WebServiceContractImpl wscImpl = WebServiceContractImpl.getInstance();
                                     ServerEnvironment se = wscImpl.getServerEnvironment();
 
+                                    // First look in the root of the deployment
                                     File appFile = new File(se.getApplicationRepositoryPath(), serviceRef.getBundleDescriptor().getApplication().getAppName());
                                     if (appFile.exists()) {
-                                        retVal = new File(appFile, serviceRef.getWsdlFileUri()).toURL();
+                                        File wsdlFile = new File(appFile, serviceRef.getWsdlFileUri());
+                                        retVal = wsdlFile.toURI().toURL();
+                                        if (!wsdlFile.exists()) {
+                                            // try the module path for example when we are in an EAR file
+                                            wsdlFile = new File(serviceRef.getBundleDescriptor().getRawModuleID(), serviceRef.getWsdlFileUri());
+                                            if (!wsdlFile.exists()) {
+                                                // finally try to load via classloader
+                                                retVal = Thread.currentThread().getContextClassLoader().getResource(serviceRef.getWsdlFileUri());
+                                            } else {
+                                                retVal = wsdlFile.toURI().toURL();
+                                            }
+                                        }
                                     } else {
                                         // Fix for 6853656 and 6868695
                                         // In case of appclients the wsdl will be in the classpath
@@ -556,7 +568,7 @@ public class WsUtil {
                                         retVal = Thread.currentThread().getContextClassLoader().getResource(serviceRef.getWsdlFileUri());
                                     }
                                 } else {
-                                    retVal = new File(serviceRef.getWsdlFileUri()).toURL();
+                                    retVal = new File(serviceRef.getWsdlFileUri()).toURI().toURL();
                                 }
                             }
                         }

